README.md

```diff
@@ -137,19 +137,23 @@ For more information on how to use QISKit, tutorial examples, and other helpful
   a GUI for interacting with real and simulated quantum computers
 * **[QISkit Python API](https://github.com/QISKit/qiskit-api-py)**, an API to use the IBM Quantum Experience in Python
 
-
 QISKit was originally developed by researchers and developers on the [IBM-Q](http://www.research.ibm.com/ibm-q/) Team at [IBM Research](http://www.research.ibm.com/), with the aim of offering a high level development kit to work with quantum computers.
 
 Visit the [IBM Q experience community](https://quantumexperience.ng.bluemix.net/qx/community) for questions and discussions on QISKit and quantum computing more broadly. If you'd like to contribute to QISKit, please take a look at our [contribution guidelines](CONTRIBUTING.rst).
 
+## Multilanguage guide
+
+* **[Korean Translation](https://github.com/QISKit/qiskit-sdk-py/tree/master/doc/ko/README-ko.md)**, Basic guide line written in Korean. 
```

Comment:
Other than the suggestions by Rudy, could you please rename the file from `doc/ko/README-ko.md` to `doc/ko/README.md` and update this link accordingly?

Reply:
@diego-plan9 I changed file name and link accordingly.